### DISCUSSION TITLE
Clarity v12 on npm but in documentation it says 5.5.4 ?

### DISCUSSION THREAD

**Discussion by akshatBais** [vmware/clarity#6356] [General] [answered]
In Clarity documentation it says v5 is the latest , but when i go to npm I also see v12 . I installed via npm install @clr/angular it installed v12 and I made changes . Is it stable ? as we have planned for production as well . Please let me know

**Comment by hippee-lee** [comment, marked as answer]
Sorry about that, v12 is the latest stable major version release (and all minors and patches). 
We bumped the version up to mirror the Angular major version so it is easier to see which version of clarity is compatible with which version of Angular going forward.

**Reply by akshatBais** [reply]
Thanks @hippee-lee for your quick response , got it . So for angular12 clarity v12 goes well . But for cds/core package i still see v5 only , why so ?

**Comment by hippee-lee** [comment]
`@cds/core` are just web-components. They are not dependent on a specific version of Angular or the `@clr/angular` components. When we decided to match `@clr/angular` versions with the Angular project we kept the `@cds/core` package on its current major version because there was no breaking change for it. Going forward we hope to have very few breaking changes for `@cds/core`. Since Angular updates to a new major version every 6 months or so and that will continue to bump the major version number of `@clr/angular` when we update it for the latest Angular versions.
